### PR TITLE
Image

### PR DESCRIPTION
**Pull request by vinc3nati**

### CODE REVIEW

File: components/Image/image.css [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+/* Rounded Image */
+.img.img-round {
+  border-radius: var(--border-rounded);
+}
+
+/* Responsive Image */
+.img.img-responsive {
+  width: 100%;
+  height: auto;
```
Comment (shrey27) [inline components/Image/image.css:17]: Just a suggestion. use object-fit:cover to maintain the aspect ratio according to changing dimensions.

**Review by shrey27 — APPROVED** [review]

File: components/Image/image.css (hunk 1 above)
Comment (vinc3nati) [reply components/Image/image.css:17]: yeah it's getting applied to the `.img` class by default

**Review by OmKakatkar — APPROVED** [review]
Good to merge.

**Review by ravikumar1002 — APPROVED** [review]
Looks Good! ready for merge

**Review by PeeyushPrashant — APPROVED** [review]
good to merge

**Comment by Shah-Arsalan** [comment]
Good to go

**Review by Shah-Arsalan — APPROVED** [review]

**Comment by Aman-Jain1996** [comment]
Looks good to merge

**Pull request merged**